src/api/sales/index.ts

```diff
@@ -19,4 +19,14 @@ export class ArcSales extends ArcAbstractAPI {
 
     return data;
   }
+
+  async getSubscriptionDetails(id: string, site: string) {
+    const response = await this.client.get(`/subscription/${id}/details`, { params: { site } });
+    return response.data;
```

Comment:
Please, add response types to sales/types.ts, and pass them as generic to 'this.client.get' call